### PR TITLE
resetBtn fix plus

### PR DESCRIPTION
**Pull request by phatpaul**
- Fix resetBtn didn't reset on ESP32.   See https://esp32.com/viewtopic.php?t=9542
- Hold resetBtn >6s wipes NVS partition to clear WiFi settings.
- Add blinky light to show WiFi status, using same timer callback as resetBtn.
- Misc fixes an improvements.

### CODE REVIEW

File: main/cgi.c [hunk 1]
```diff
@@ -34,7 +34,7 @@ CgiStatus ICACHE_FLASH_ATTR cgiLed(HttpdConnData *connData) {
 	len=httpdFindArg(connData->post.buff, "led", buff, sizeof(buff));
 	if (len!=0) {
 		currLedState=atoi(buff);
-		ioLed(currLedState);
+		gpio_set_level(LED_CGI, currLedState);
```
Comment (chmorgan) [inline main/cgi.c]: should this be using an ioLed() call instead, as you've introduced those abstractions for the led control?

Comment (phatpaul) [reply main/cgi.c]: I know it is inconsistent.  I didn't like the abstraction that was there as it seemed unnecessary to introduce another variable to index the LEDs.
I'll adjust to make it consistent by letting ioLed take a GPIO number as input.

**Comment by phatpaul** [comment]
Yeah, maybe I get carried away.  I'll try to keep the additions to a minimum.
 I realize this was just supposed to be a test project for libesphttpd, but I used it as a starting point for my project.  I see it as a framework or reference to start a project which uses an embedded webserver as its primary UI.

I'd be happy to help as a team member.

**Comment by chmorgan** [comment]
@phatpaul oh I think the changes are probably great to have, especially if they are documented so people can understand how to make use of the project. Do they work with the standard esp32-dev board?

I think separating out the eclipse file changes is a good start.

Let me add you to this and the other repository teams right now.

**Comment by phatpaul** [comment]
@chmorgan I removed the eclipse stuff and cleaned up the LED stuff.  Are you ok if I merge?

**Comment by chmorgan** [comment]
@phatpaul looks good man

**Pull request merged**